pyproject.toml

```diff
@@ -151,10 +148,12 @@ xarray = "xr"
 [tool.pytest.ini_options]
 # These follow standard library warnings filters syntax.  See more here:
 # https://docs.python.org/3/library/warnings.html#describing-warning-filters
-addopts = "--maxfail=5 --durations=10 -rXxs"
+addopts = ["--maxfail=5",  "--durations=10",  "-ra", "--strict-markers", "--strict-config"]
```

Comment:
@brisvag Asked changes comes from using `--strict-config` here. 

It causes pytest to fail if some option is not recognized. It makes harder to configure optional plugins, but save from making typos in configuration options.

Here issue in pytest-pystack https://github.com/bloomberg/pytest-pystack/issues/9